.github/workflows/build-all.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           ghc: '9.4.8'
 
         - os: macos-12
```

Comment:
Let's add a comment that macos-12 was the last version of the macos runner that uses intel